apps/distgo/cmd/publish/github_publish.go

```diff
@@ -23,11 +23,9 @@ import (
 	"os"
 	"path"
 	"path/filepath"
-	"reflect"
-	"strings"
 
 	"github.com/google/go-github/github"
-	"github.com/google/go-querystring/query"
```

Comment:
OK tried locally, and looks like removing this from the vendor directory causes `extimport` to flag it because it is a real error (it's transitively used).

For now, we'll have to add to the `novendor` whitelist in `checks.yml`. This is issue https://github.com/palantir/checks/issues/16.